### ISSUE TITLE
wsl2 ubuntu 20.04 markdownpreview无法启动chrome

### ISSUE THREAD

**Issue by sectum-sempra** [iamcco/markdown-preview.nvim#309]
:mess显示结果如下
[markdown-preview.nvim]: Can not open browser by using cmd.exe command

init.vim里对应部分设置如下:
let g:mkdp_preview_options = {                                                               \ 'mkit': {},
      \ 'katex': {},
      \ 'uml': {},
      \ 'maid': {},
      \ 'disable_sync_scroll': 0,
      \ 'sync_scroll_type': 'middle',
      \ 'hide_yaml_meta': 1,
      \ 'sequence_diagrams': {},
      \ 'flowchart_diagrams': {},
      \ 'content_editable': v:false,
      \ 'disable_filename': 0
      \ }
  let g:mkdp_browser = 'chrome'
  let g:mkdp_filetypes = ['markdown','rmd']
  let g:mkdp_auto_start = 1
  let g:mkdp_command_for_global = 1
  let g:mkdp_echo_preview_url = 1
  let g:mkdp_open_to_the_world = 1

另外chrome已经添加到user和system的path里, 在cmd里输入chrome可以正常打开chrome

**Comment by iamcco** [comment]
你用 `wsl` 关键字搜下 issue，记得之前有人遇到过

**Comment by Qiguang** [comment]
After several hours researching, I found the reason of this issue (at least for my situation).

**TL;DR**

Just try launch nvim by this command: `sudo env "PATH=$PATH" nvim`, then check if the issue is solved.

---

As this plugin launches Chorme using `cmd.exe` , So test if you can launch cmd.exe from your WSL console directly. If the environment of your WSL is correct, just type`cmd.exe`

then, cmd window will shown.

If it cannot be launched, check your $PATH variable. [Reference](https://www.bleepingcomputer.com/news/microsoft/how-to-run-windows-10-programs-in-a-wsl-linux-shell/)

You can launch cmd.exe in WSL console means you can launch it in nvim. ([how to run shell command in vim](https://superuser.com/questions/285500/how-to-run-unix-commands-from-within-vim))

But I tried to launch cmd.exe in nvim by command `:!cmd.exe`, error prompted `/bin/bash: cmd.exe: command not found`

After researching, I found this is because I launched nvim by command `sudo nvim`, but  `sudo`  changed my default `$PATH` variable of shell, so `cmd.exe` cannot be found. [Reference](https://stackoverflow.com/questions/257616/why-does-sudo-change-the-path)

To avoid this, run nvim by using `sudo env "PATH=$PATH" nvim`  which keeps the default $PATH, then all issues disappears.

@sectum-sempra 你试一下在nvim里直接启动cmd.exe能启动成功吗？不能的话，可能是PATH变量的问题。

**Comment by isanasan** [comment]
This problem is resolved.

Just try this command
>export PATH=$PATH:/mnt/c/Windows/System32/

**Comment by P2LOVE** [comment]
Also:
 `/etc/wsl.conf`
```
[interop]
enabled = true
appendWindowsPath = true
```
then close WSL terminal

Run `cmd.exe`
`wsl --shutdown`

Now open WSL again and $PATH is merged with Windows
